ashmanmalik/account-verification
# chore: Update Dependencies and Fix ESLint Warnings

### PR Title
Update Dependencies and Fix ESLint Warnings 🛠️✨

### Description
This pull request updates various dependencies and addresses ESLint warnings related to React hooks. 

### Changes Made
- **Updated Dependencies:**
  - Bumped versions of `@headlessui/react`, `@vercel/analytics`, `axios`, `react`, `react-dom`, and others.
  - Updated `cypress`, `eslint`, and `tailwindcss` to their latest versions.
  
- **Fixed ESLint Warnings:**
  - Added missing dependencies in `useEffect` hooks in:
    - `AccountVerificationFormStep0SignUp.js`
    - `AccountVerificationFormStep3LoadingSteps.js`

### Why This Matters
- Keeping dependencies up-to-date improves security and performance.
- Addressing ESLint warnings enhances code quality and helps prevent potential bugs.

### Testing
- Run `next build` to ensure there are no build issues.
- Verify functionality in the affected components to ensure everything works as expected.

### Checklist
- [ ] Updated relevant documentation (if needed)
